### PR TITLE
Changed the phrasing

### LINKED ISSUE #24918: Phrasing is misleading
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

### What specific section or headline is this issue about?

Active learning: Implementing your own responsive images

### What information was incorrect, unhelpful, or incomplete?

Use srcset/size to create a resolution switcher example, either to serve the same size image at different resolutions, or different image sizes at different viewport widths.

### What did you expect to see?

Use srcset/size to create a resolution switcher example, either to serve the same size image at different resolutions depending on the device resolution, or to serve different image sizes depending on the viewport widths.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

The word "at" means something different in each occurrence:

1. "same size image at different resolution": Here, "at" means "with".
2. "different image sizes at different viewport widths": Here, "at" means" depending on the".

This is unclear to the reader, who could be stuck wondering what they're being asked to do.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/html/multimedia_and_embedding/responsive_images`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md
* Last commit: https://github.com/mdn/content/commit/8e2641ebe076ab89299c77a51ece882de4ba5efb
* Document last modified: 2023-02-23T23:55:46.000Z

</details>

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I changed the phrasing in step 5 by implementing @joelc0193's suggestion.
### Motivation

Doing these changes make the directions clearer 
### Related issues and pull requests
Fixes [#24918](https://github.com/mdn/content/issues/24918)

### CODE REVIEW

File: files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ For this active learning, we're expecting you to be brave and do it alone, mostl
 2. Find a nice wide screen landscape image with some kind of detail contained in it somewhere. Create a web-sized version of it using a graphics editor, then crop it to show a smaller part that zooms in on the detail, and create a second image (about 480px wide is good for this).
 3. Use the `<picture>` element to implement an art direction picture switcher!
 4. Create multiple image files of different sizes, each showing the same picture.
-5. Use `srcset`/`size` to create a resolution switcher example, either to serve the same size image at different resolutions, or different image sizes at different viewport widths.
+5. Use srcset/size to create a resolution switcher example, either to serve the same size image at different resolutions depending on the device resolution, or to serve different image sizes depending on the viewport widths.
```
Comment (estelle) [inline files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md:210]:
```suggestion
5. Use `srcset`/`size` to create a resolution switcher example, either to serve the same size image at different resolutions depending on the device resolution or to serve different image sizes depending on the viewport widths.
```

The back tics turn the content between them into `<code>`. We always want to codify attributes, properties, element names, values, etc.

**Review by estelle — COMMENTED** [review]
thanks. Just one fix, and we should be good to go.

**Comment by CBID2** [comment]
> thanks. Just one fix, and we should be good to go.

Done @estelle!😁

File: files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md [hunk 2]
```diff
@@ -207,7 +207,7 @@ For this active learning, we're expecting you to be brave and do it alone, mostl
 2. Find a nice wide screen landscape image with some kind of detail contained in it somewhere. Create a web-sized version of it using a graphics editor, then crop it to show a smaller part that zooms in on the detail, and create a second image (about 480px wide is good for this).
 3. Use the `<picture>` element to implement an art direction picture switcher!
 4. Create multiple image files of different sizes, each showing the same picture.
-5. Use `srcset`/`size` to create a resolution switcher example, either to serve the same size image at different resolutions, or different image sizes at different viewport widths.
+5. Use `srcset`/`size` to create a resolution switcher example, either to serve the same size image at different resolutions depending on the device resolution or to serve different image sizes depending on the viewport widths.
```
Comment (dipikabh) [inline files/en-us/learn/html/multimedia_and_embedding/responsive_images/index.md:210]: Also, it should be `sizes`

```suggestion
5. Use `srcset`/`sizes` to create a resolution switcher example, either to serve the same size image at different resolutions depending on the device resolution or to serve different image sizes depending on the viewport widths.
```

I do feel that the original sentence was okay and clear

**Review by dipikabh — CHANGES_REQUESTED** [review]
A tiny nit

**Comment by CBID2** [comment]
> A tiny nit

Made the changes! 😊

**Review by dipikabh — APPROVED** [review]
Thank you!

**Pull request merged**